runtime/centrifuge/src/weights/pallet_oracle_collection.rs

```diff
@@ -90,14 +90,14 @@ impl<T: frame_system::Config> pallet_oracle_collection::WeightInfo for WeightInf
 	fn update_collection(n: u32, m: u32, ) -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `0 + m * (326 ±0) + n * (5851 ±0)`
-		//  Estimated: `16920 + m * (6039 ±164) + n * (100600 ±3_323)`
-		// Minimum execution time: 129_342_000 picoseconds.
-		Weight::from_parts(130_825_000, 0)
+		//  Estimated: `16920 + m * (6039 ±0) + n * (100600 ±3_323)`
+		// Minimum execution time: 127_009_000 picoseconds.
+		Weight::from_parts(128_011_000, 0)
```

Comment:
Similar-smaller weights with the wasm executor? I would expect more weight 🤔

Reply:
Ok, I would expect the benchmarking to take longer in wasm, because is slower than native. But maybe somehow the benchmarking is done in the same environment before and now